e2e/utils/selectors.utils.ts

```diff
@@ -0,0 +1,22 @@
+import { RECEIVE_TOKENS_MODAL_ADDRESS_LABEL } from '$lib/constants/test-ids.constants';
+
+/**
+ * Generates a selector that can be used to query an inner element by specifying its parent
+ */
+export const getNestedSelector = ({
+	parentSelector,
+	innerSelector
+}: {
+	parentSelector: string;
+	innerSelector: string;
+}) => `[data-tid="${parentSelector}"] >> [data-tid="${innerSelector}"]`;
+
+export const getReceiveTokensModalAddressLabelSelectorBySection = ({
```

Comment:
if you want you can make it less verbose, removing the `BySection` part?

Reply:
👍